Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.yml

```diff
@@ -1347,11 +1353,135 @@ script:
+      isArray: true
+      name: updates
+      required: true
+    - description: Enrollment info in dict format. If provided, the script will not make another API call to get the enrollmont info.             if not, another API call will be issued to retrieve the Enrollment info.
+      isArray: true
+      name: enrollment
+    - description: All pending changes to be cancelled when updating an enrollment.
+      name: allow_cancel_pending_changes
+      auto: PREDEFINED
+      defaultValue: 'true'
+      predefined:
+      - 'true'
+      - 'false'
+    - description: Bypass staging and push meta_data updates directly to production network. Current change will also be updated with the same changes.
+      name: allow_staging_bypass
+      auto: PREDEFINED
+      defaultValue: 'true'
+      predefined:
+      - 'true'
+      - 'false'
+    - description: "Don't deploy after this date (UTC). Sample: 2021-01-31T00:00:00.000Z."
+      name: deploy_not_after
+    - description: "Don't deploy before this date (UTC). Sample: 2021-01-31T00:00:00.000Z."
+      name: deploy_not_before
+    - description: Force certificate renewal for Enrollment.
+      name: force_renewal
+      auto: PREDEFINED
+      defaultValue: 'false'
+      predefined:
+      - 'true'
+      - 'false'
+    - description: CPS will automatically start a Change to renew certificates in time before they expire.
+      name: renewal_date_check_override
+      auto: PREDEFINED
+      defaultValue: 'true'
+      predefined:
+      - 'true'
+      - 'false'
+    - description: Applicable for Third Party Dual Stack Enrollments, allows to update missing certificate. Option supported from v10.
+      name: allow_missing_certificate_addition
+      auto: PREDEFINED
+      defaultValue: 'false'
+      predefined:
+      - 'true'
+      - 'false'
+    description: 'Updates an enrollment with changes. Response type will vary depending on the type and impact of change. For example, changing SANs list may return HTTP 202 Accepted since the operation require a new certificate and network deployment operations, and thus cannot be completed without a change. On the contrary, for example a Technical Contact name change may return HTTP 200 OK assuming there are no active change and when the operation does not require a new certificate. Reference: https://techdocs.akamai.com/cps/reference/put-enrollment  NOTES: Depending on the type of the modification, additional steps might be required to complete the update. These additional steps could be carrying out a "renew" change by resubmitting the CSR, acknowleging the warnings raised then waiting for the certificate to be deployed into PRODUCTION. However these additional steps are not included in this command. User needs to conduct those steps once the update command is completed.'
```

Comment:
```suggestion
    description: 'Updates an enrollment with changes. Response type will vary depending on the type and impact of change. For example, changing SANs list may return HTTP 202 Accepted since the operation requires new certificate and network deployment operations, and thus cannot be completed without a change. On the contrary, for example a Technical Contact name change may return HTTP 200 OK assuming there are no active changes and the operation does not require a new certificate. Reference: https://techdocs.akamai.com/cps/reference/put-enrollment  Note: Depending on the type of the modification, additional steps might be required to complete the update. These additional steps could be carrying out a "renew" change by resubmitting the CSR, acknowledging the warnings raised then waiting for the certificate to be deployed into Production. However, these additional steps are not included in this command. You need to perform those steps once the update command is completed.'
```